### PR TITLE
Secure Practice MailRisk Solution

### PR DESCRIPTION
**Pull request by ingin97**
Change(s):
   - Add sample data
   - Add logo
   - Add data connector json file
   - Add custom logs table schema

   Reason for Change(s):
   - Initial commit for new MailRisk Solution. Step 1 from the Guide to building Microsoft Sentinel solutions. 

   Version Updated:
   - N/A

   Testing Completed:
   - No/Need Help

   Checked that the validations are passing and have addressed any issues that are present:
   -  No/Need Help

   Question:
   -  Is it okay to to only add the Data connector in the first release, or should we also include Workbooks, Analytic Rules, Hunting queries, Playbooks, Parser?

### CODE REVIEW

**Comment by vmanojreddy** [comment]
Hi @ingin97 In the check Sample Data validator is failing due to Email must be sanitized@sanitized.com

**Comment by vmanojreddy** [comment]
Hi @ingin97 In the check  Data connector validator is failing due to does not match with Azure Function Connector Template.
Please refer template : https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/DataConnector_API_AzureFunctionApp_template.json.

**Comment by ingin97** [comment]
@vmanojreddy What should we do to proceed?

File: Solutions/MailRisk/Data Connectors/SecurePractice_MailRisk.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            "description": "Use this method for automated deployment of the MailRisk data connector using an ARM Template.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fgithub.com%2Fsecurepractice%2Fmailrisk-sentinel-connector%2Fblob%2Fmaster%2Fazuredeploy.json)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n3. Enter the **Workspace ID**, **Workspace Key**, **Secure Practice API Key**, **Secure Practice API Secret** \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**.\n5. Click **Purchase** to deploy."
+        },
+        {
+            "title": "Manual deployment",
+            "description": "In the open source repository on [GitHub](https://github.com/securepractice/mailrisk-sentinel-connector) you can find instructions for how to manually deploy the data connector."
+        }
+    ],
+    "metadata": {
+        "id": "c9c97ce4-2093-466c-846e-49be58a39197",
+        "version": "0.1",
```
Comment (v-sabiraj) [inline Solutions/MailRisk/Data Connectors/SecurePractice_MailRisk.json:123]: @ingin97, please update the version in correct format. e.g 1.0.0, thanks.

File: Solutions/MailRisk/Data Connectors/SecurePractice_MailRisk.json [hunk 2]
```diff
@@ -0,0 +1,140 @@
+{
+    "id": "SecurePractice_MailRisk",
+    "title": "MailRisk by Secure Practice",
+    "publisher": "Secure Practice",
+    "descriptionMarkdown": "Data connector to push emails from MailRisk into Azure Sentinel Log Analytics.",
```
Comment (v-sabiraj) [inline Solutions/MailRisk/Data Connectors/SecurePractice_MailRisk.json:5]: @ingin97, please change Azure Sentinel to Microsoft Sentinel,

File: Solutions/MailRisk/Data Connectors/SecurePractice_MailRisk.json [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            {
+                "name": "API credentials",
+                "description": "Your Secure Practice API key pair is also needed, which are created in the [settings in the admin portal](https://manage.securepractice.co/settings/security). If you have lost your API secret, you can generate a new key pair (WARNING: Any other integrations using the old key pair will stop working)."
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description": ">**NOTE:** This connector uses Azure Functions to connect to the Secure Practice API to push logs into Azure Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
```
Comment (v-sabiraj) [inline Solutions/MailRisk/Data Connectors/SecurePractice_MailRisk.json:86]: @ingin97,please change Azure Sentinel to Microsoft Sentinel, thanks.

**Comment by ingin97** [comment]
@vmanojreddy What is left before it can get approved?

**Review by vmanojreddy — APPROVED** [review]

**Review by v-sabiraj — APPROVED** [review]

**Review by NikTripathi — APPROVED** [review]

**Pull request merged**